src/app/public/modules/code-examples/code-example.component.ts

```diff
@@ -42,8 +56,43 @@ export class SkyDocsCodeExampleComponent {
 
   /**
    * Specifies if the editor service should show the example in modern theme.
+   * The value will be gleaned from `SkyThemeService` first, then fallback to 'default' if the service is not found.
    */
   @Input()
-  public theme: SkyDocsCodeExampleTheme = SkyDocsCodeExampleTheme.Default;
+  public set theme(value: SkyDocsCodeExampleTheme) {
+    this._theme = value;
+  }
+
+  public get theme(): SkyDocsCodeExampleTheme {
+    return this._theme || SkyDocsCodeExampleTheme.Default;
+  }
+
+  private ngUnsubscribe = new Subject<boolean>();
```

Comment:
Non-blocking feedback, but the `Subject` can be `void`, especially since the `.next()` call is empty.

Reply:
Done.